xml/HumanitarianClusters.xml

```diff
@@ -0,0 +1,194 @@
+<codelist xmlns:budget-alignment="http://iatistandard.org/activity-standard/overview/country-budget-alignment/" name="Humanitarian Clusters" xml:lang="en" category-codelist="HumanitarianClusters" complete="1" embedded="0">
+    <metadata>
+        <name>
+            <narrative>Global Coordination Groups</narrative>
+            <narrative xml:lang="fr">Secteur du CAD à 5 chiffres</narrative>
+        </name>
+        <url>https://data.humdata.org/dataset/global-coordination-groups-beta</url>
+        <category>
+            <narrative>CodeforIATImaintained</narrative>
+        </category>
+    </metadata>
+    <codelist-items>
+        <codelist-item status="active">
+            <code>4</code>
+            <name>
+                <narrative>Emergency Shelter and NFI</narrative>
+                <narrative xml:lang="fr">Abris d'urgence et NFI</narrative>
+            </name>
+            <description>
+                <narrative>Shelter and Non-Food Items includes provision of shelter materials and non-food household item packages. The theme also covers Camp Coordination and Camp Management. Long-term/permanent reconstruction/rebuilding of housing is covered under "Recovery and Reconstruction."</narrative>
+                <narrative xml:lang="fr">Le volet Logement et articles non alimentaires comprend la fourniture de matériaux de construction et d'ensembles d'articles ménagers non alimentaires. Le thème couvre également la coordination et la gestion des camps. La reconstruction/reconstruction à long terme/permanente des logements est couverte par le thème "Relèvement et reconstruction".</narrative>
+            </description>
+            <category>Cluster</category>
+        </codelist-item>
+        <codelist-item status="active">
+                    <code>1</code>
```

Comment:
Would be good to pretty print, as there are some whitespace issues. I know this sounds very pedantic… But it’s useful if we do end up scraping and generating this automatically in the future.

Reply:
I’ve updated this so that it’s pretty printed now.